docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -781,6 +783,14 @@ If enabled, makes the image read-only.
 
 If ReadOnly is set to `true`, mount a read-write tmpfs on /dev, /dev/shm, /run, /tmp, and /var/tmp.
 
+### `Retry=`
+
+Number of times to retry the image pull when a HTTP error occurs. Passed as `--retry` to podman.
```

Comment:
Just to align the wording, instead of "Passed as `XXX`" please use "Equivalent to the Podman `XXX` option"